### PR TITLE
SDK-17 Remove legacy Word .doc format

### PR DESCRIPTION
**Pull request by zhaveriane**

### CODE REVIEW

File: spec/lib/rev/get_attachment_content_spec.rb [hunk 1]
```diff
@@ -62,12 +62,11 @@
     action.must_raise Rev::NotFoundError
   end
 
-  # requesting conversion from pdf to :docx from order TC0263917003
```
Comment (BlakeHenry) [inline spec/lib/rev/get_attachment_content_spec.rb]: Optional: If you look at line 16 if you can see we wrap a couple tests in a 'success' block which groups the tests which makes it a little more organized. The tests that test failure are currently not wrapped in a block like that. Consider wrapping these two tests as a way to clean up this code a little.

`describe 'not allowed' do`
tests
`end`

Comment (BlakeHenry) [reply spec/lib/rev/get_attachment_content_spec.rb]: screen shot 2018-07-10 at 12 16 23 pm

Makes the tests organized when running rake test

**Comment by bryanroscoe** [comment]
I assume we need to publish a new version so we should change the gem version for this sdk?

**Comment by BlakeHenry** [comment]
Heres a guide for pushing the new ruby gem version. I think we can bump it x.x.1 since this is a bug fix.
https://guides.rubygems.org/make-your-own-gem/

Heres some info on versioning if you're interested: https://guides.rubygems.org/patterns/#semantic-versioning

File: spec/lib/rev/get_attachment_content_spec.rb (hunk 1 above)
Comment (EfrainReyes) [inline spec/lib/rev/get_attachment_content_spec.rb]: This PDF-to-docx test was confusing, I initially thought there was a valid test case we were accidentally removing, but it looks like it's just mocked behavior.

**Pull request merged**